src/commands/up.rs

```diff
@@ -66,10 +70,21 @@ pub struct UpCommand {
         name = BINDLE_PASSWORD,
         long = "bindle-password",
         env = BINDLE_PASSWORD,
-        requires = BINDLE_USERNAME
+        requires = BINDLE_USERNAME,
+        hide = true,
     )]
     pub bindle_password: Option<String>,
 
+    /// Reference to run the application from a container registry.
```

Comment:
Given our messaging about Wasm and containers, saying we run things from a 'container' registry risks muddying the waters.  "Oh, does Spin run containers as well as Wasm, or is this running it in a container?"

Reply:
Changed to simply `... from a registry`. Thanks!